### DISCUSSION TITLE
using-azure-storage-queues-with-azure-functions-and-queuetrigger/

### DISCUSSION THREAD

**Discussion by giscus** [andrewlock/blog-comments#432] [Announcements]
# using-azure-storage-queues-with-azure-functions-and-queuetrigger/

In this post I discuss using Azure Storage Queue as a message queue and how you can process Storage Queue messages using an Azure Functions app

https://andrewlock.net/using-azure-storage-queues-with-azure-functions-and-queuetrigger/

**Comment by jonsagara** [comment]
Hi Andrew,

Have you experience not being able to observe the `CancellationToken` when running in the isolated model? I can't get it to work correctly in either .NET 6 or 8, but it works fine in the in-process model. Unfortunately, I also can't get a response from the Azure Functions team, so I'm wondering if anyone else is experiencing the issue.

https://github.com/Azure/azure-functions-dotnet-worker/issues/2510

Great article, and thank you.

Jon Sagara

**Comment by 1RedOne** [comment]
Great post and really inspired me.  I agree that the docs were quite confusing on how to configure this to use managed identities as well!  

I got it working and posted a guide on my own site over here:  https://www.foxdeploy.com/blog/how-to-migrate-an-azure-function-app-to-use-a-managed-identity.html
